doc/source/user_guide/groupby.rst

```diff
@@ -1049,16 +1049,14 @@ for both ``aggregate`` and ``transform`` in many standard use cases. However,
 
 The dimension of the returned result can also change:
 
-.. ipython::
-
-    In [8]: grouped = df.groupby('A')['C']
+.. ipython:: python
```

Comment:
can you add some commentary on groupby_keys=True/False here (maybe show both examples?